### PR TITLE
Local file removal

### PR DESCRIPTION
**Pull request by rfum**
# Contributing to the AWS Control Tower Account Factory for Terraform

Thank you for your interest in contributing to the AWS Control Tower Account Factory for Terraform.

At this time, we are not accepting contributions. If contributions are accepted in the future, the AWS Control Tower Account Factory for Terraform is released under the [Apache license](http://aws.amazon.com/apache2.0/) and any code submitted will be released under that license.

If you have a feature request, please create an issue using the Feature Request template, thanks!

### CODE REVIEW

**Comment by monteiro-renato-cog** [comment]
README needs to be updated.
It might also be a good idea to add to the README the reason why we are forking the repo and which changes have been made.

**Comment by rfum** [comment]
> README needs to be updated. It might also be a good idea to add to the README the reason why we are forking the repo and which changes have been made.

I'm not yet done with the repo I'm planning to write readme when everything is ok on our end.

File: modules/aft-code-repositories/codebuild.tf [hunk 1]
```diff
@@ -46,7 +46,7 @@ resource "aws_codebuild_project" "account_request" {
 
   source {
     type      = "CODEPIPELINE"
-    buildspec = data.local_file.account_request_buildspec.content
+    buildspec = var.account_provisioning_customizations_buildspec
```
Comment (monteiro-renato-cog) [inline modules/aft-code-repositories/codebuild.tf]: This variable and the variable on line 102 are mixed up.

File: variables.tf [hunk 2]
```diff
@@ -387,3 +387,635 @@ variable "aft_metrics_reporting" {
...
+  type    = string
+  default = "1.11.1"
+}
+
+variable "python_version" {
+  type    = string
+  default = "3.11"
+}
+
+variable "ct_aft_account_provisioning_customizations" {
```
Comment (monteiro-renato-cog) [inline variables.tf:405]: This looks a bit ugly.
Have we tried > https://developer.hashicorp.com/terraform/language/functions/file ?
Do we know why upstream chose to use local_file?

**Review by kondanta — APPROVED** [review]

**Pull request merged**